File: samples/python/hello_world.py
"""Testing Script for python to validate if all language features are working.

Copyright (c) 2024 Radiant Science Inc.
"""

from samples.python.hello_lib import get_proto


def main() -> None:
    sample_message = get_proto()
    print(sample_message)

    print("Hello World")


if __name__ == "__main__":
    main()

File: samples/rust/hello_world.rs
// Simple Test for Rust.
// Copyright (c) 2024 Radiant Science Inc.

// import hello_lib
use samples::rust::hello_lib::get_proto;

fn main() -> anyhow::Result<()> {
    // create proto msg
    let proto = get_proto();
    // print the message
    println!("{:?}", proto);
    println!("Hello, world!");
    Ok(())
}
